Mathlib/Data/Set/Finite/Parity.lean

```diff
@@ -0,0 +1,47 @@
+/-
+Copyright (c) 2024 Pim Otte. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Pim Otte
+-/
+import Mathlib.Data.Set.Card
+import Mathlib.Data.Set.Finite
+
+/-!
+# Finite sets and parity of their cardinality
+
+This file contains lemmas concerning finite sets and parity of their cardinality.
```

Comment:
```suggestion
This file contains lemmas concerning finite sets and the parity of their cardinality.
```
Perhaps this could go a bit more in depth about the scope of the file? Or, if these are the two only theorems you plan on adding, maybe they can be moved elsewhere? Perhaps to the same file of the rest of your proof.

Reply:
The vast majority of the rest of the proof would be in Combinatorics.SimpleGraph, so if you think a separate file is overkill, I'd say Data.Set.Card is the most likely alternative.

Reply:
In refactoring the code became even smaller, so I moved it to Data.Set.Card